$OpenBSD: patch-src_platform_sdl_CMakeLists_txt,v 1.2 2015/07/19 22:24:11 bentley Exp $
Pixman code is busted, workaround suggested by upstream.
Only an issue on SDL 1.2 platforms like OpenBSD.
--- src/platform/sdl/CMakeLists.txt.orig	Tue Apr  7 01:54:02 2015
+++ src/platform/sdl/CMakeLists.txt	Tue Apr  7 01:54:02 2015
@@ -17,7 +17,6 @@ if(SDL_VERSION EQUAL "1.2" OR NOT SDL2_FOUND)
 	find_package(SDL 1.2)
 	set(SDL_VERSION "1.2" PARENT_SCOPE)
 	set(SDL_VERSION_DEBIAN "1.2debian")
-	set(USE_PIXMAN ON)
 endif()
 
 if (NOT SDL2_FOUND AND NOT SDL_FOUND)
